### PR TITLE
Update Arrays and Query Expression BBEs

### PR DESCRIPTION
**Pull request by lasinicl**
## Automation tests
 - Unit tests 
   > Code coverage information
 - Integration tests
   > Details about the test cases and coverage

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes/no
 - Ran FindSecurityBugs plugin and verified report? yes/no
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes/no

### CODE REVIEW

File: examples/arrays/arrays.bal [hunk 1]
```diff
@@ -26,6 +27,20 @@ public function main() {
         });
     io:println("Doubled: ", doubled);
 
+    // An array can be sorted using `.sort()`. `array:DESCENDING` is the `direction` in which to sort.
```
Comment (praneesha) [inline examples/arrays/arrays.bal:30]:
```suggestion
    // An array can be sorted using `.sort()`. The `array:DESCENDING` is the `direction` in which sort should be done.
```

File: examples/arrays/arrays.bal [hunk 2]
```diff
@@ -26,6 +27,20 @@ public function main() {
...
+    // The default sort direction is `ASCENDING`. The `isolated function` returns a `string[]` for each
+    // member which is used as a `key` to sort the members. If the member type of the array
```
Comment (praneesha) [inline examples/arrays/arrays.bal:32]:
```suggestion
    // member, which is used as a `key` to sort the members. If the member type of the array
```

File: examples/arrays/arrays.bal [hunk 3]
```diff
@@ -26,6 +27,20 @@ public function main() {
...
+    // is not sorted, then the `key` function must be specified.
+    // It returns an `array` consisting of the members of `b` in sorted order.
```
Comment (praneesha) [inline examples/arrays/arrays.bal:34]:
```suggestion
    // It returns an `array` consisting of the members of `b` in the sorted order.
```

File: examples/query-expression/query_expression.bal [hunk 4]
```diff
@@ -36,6 +36,9 @@ public function main() {
        //The `let` clause binds the variables.
        let string degreeName = "Bachelor of Medicine",
        int graduationYear = calGraduationYear(student.intakeYear)
+       //The `order by` clause sorts the output items based on the given `order-key` and `order-direction`.
+       //The `order-key` must be an ordered type. The `order-direction` is `ascending` if not explicitly specified.
```
Comment (praneesha) [inline examples/query-expression/query_expression.bal:40]:
```suggestion
       //The `order-key` must be an ordered type. The `order-direction` is `ascending` if not specified explicitly.
```

**Review by praneesha — APPROVED** [review]

**Comment by niveathika** [comment]
@lasinicl Ensure build is passing before merging this

**Pull request merged**